packages/payment/src/services/payment-module.ts

```diff
@@ -271,4 +279,29 @@ export default class PaymentModuleService implements IPaymentModuleService {
   ): Promise<PaymentCollectionDTO> {
     throw new Error("Method not implemented.")
   }
+
+  private async createProvidersOnLoad() {
```

Comment:
todo: Should use payment provider service from #6308

Reply:
should we move that to a loader instead of the hook?